test/loki_service.cc

```diff
@@ -94,87 +94,88 @@ const std::vector<http_request_t> valhalla_requests{
+                                                                       {400,
+                                                                        R"({"error_code":150,"error":"Exceeded max locations: 20","status_code":400,"status":"Bad Request"})"},
+                                                                       {400,
+                                                                        R"({"error_code":120,"error":"Insufficient number of locations provided","status_code":400,"status":"Bad Request"})"},
+                                                                       {400,
+                                                                        R"({"error_code":112,"error":"Insufficiently specified required parameter 'locations' or 'sources & targets'","status_code":400,"status":"Bad Request"})"},
+                                                                       {400,
+                                                                        R"({"error_code":120,"error":"Insufficient number of locations provided","status_code":400,"status":"Bad Request"})"},
+                                                                       {400,
+                                                                        R"({"error_code":130,"error":"Failed to parse location","status_code":400,"status":"Bad Request"})"},
+                                                                       {400,
+                                                                        R"({"error_code":130,"error":"Failed to parse location","status_code":400,"status":"Bad Request"})"},
+                                                                       {400,
+                                                                        R"({"error_code":124,"error":"No edge\/node costing provided","status_code":400,"status":"Bad Request"})"},
+                                                                       {400,
+                                                                        R"({"error_code":131,"error":"Failed to parse source","status_code":400,"status":"Bad Request"})"},
+                                                                       {400,
+                                                                        R"({"error_code":132,"error":"Failed to parse target","status_code":400,"status":"Bad Request"})"},
+                                                                       {400,
+                                                                        R"({"error_code":157,"error":"Exceeded max avoid locations: 0","status_code":400,"status":"Bad Request"})"},
+                                                                       {400,
+                                                                        R"({"error_code":153,"error":"Too many shape points: (102). The best paths shape limit is 100","status_code":400,"status":"Bad Request"})"}};
```

Comment:
extra comma will fix whitespace change
```suggestion
                                                                        R"({"error_code":153,"error":"Too many shape points: (102). The best paths shape limit is 100","status_code":400,"status":"Bad Request"})"},};
```